pkg/controllers/hypershiftdeployment_controller.go

```diff
@@ -80,15 +88,16 @@ func (r *HypershiftDeploymentReconciler) Reconcile(ctx context.Context, req ctrl
 	log := r.Log
 	r.ctx = ctx
 
-	log.Info("Reconcile...")
+	log.Info(fmt.Sprintf("Reconcile: %s", req))
+	defer log.Info(fmt.Sprintf("Reconcile: %s Done", req))
 
 	var hyd hypdeployment.HypershiftDeployment
 	if err := r.Get(ctx, req.NamespacedName, &hyd); err != nil {
 		log.V(2).Info("Resource deleted")
 		return ctrl.Result{}, nil
 	}
 
-	r.Log.Info("Reconciling")
+	log.Info("Reconciling")
```

Comment:
I shouldn't have added this second Reconciling since we state it 10 lines above.